tox.ini

```diff
@@ -1,7 +1,8 @@
 [tox]
 minversion = 2.4
+envlist = py{2.7,3.6}
```

Comment:
Fair enough; I guess this requires appending `- 3.6` to `python:` in `.travis.yml` then?